config/externalname.go

```diff
@@ -1354,6 +1354,13 @@ var ExternalNameConfigs = map[string]config.ExternalName{
 	// imported using the standards subscription ARN:
 	// arn:aws:securityhub:eu-west-1:123456789012:subscription/pci-dss/v/3.2.1
 	"aws_securityhub_standards_subscription": FormattedIdentifierFromProvider("", "standards_arn"),
+
+	// cloudformation
+	//
+	// arn:aws:cloudformation:us-west-1:123456789123:stack/networking-stack/1e691240-6f2c-11ed-8f91-06094dc221f3
+	"aws_cloudformation_stack": TemplatedStringAsIdentifierWithNoName("arn:aws:cloudformation:{{ .parameters.region }}:{{ .client_metadata.account_id }}:stack/{{ .parameters.name }}/{{ .external_name }}"),
```

Comment:
Could you please leave a code comment explaining that the identifier for the resource turned out to be an ARN, instead of being its name?

When you look at the Terraform [docs](https://registry.terraform.io/providers/hashicorp/aws/4.15.1/docs/resources/cloudformation_stack#import), it looks like we can use `config.NameAsIdentifier` here, but if I remember correctly, this did not work. So, leaving a comment here would help the maintainers.

Reply:
Sure. I added a comment, could you please take a look at it. If it's ok I won't rerun tests - change doesn't require it, right?

Reply:
And yes, you're right with `config.NameAsIdentifie` this resource doesn't work. You helped me configure it before.)